package.json

```diff
@@ -3,11 +3,13 @@
 	"version": "2.2.0",
 	"type": "module",
 	"scripts": {
-		"openapi": "bunx --bun openapi-typescript ./openapi.json -o ./src/lib/schema.gen.d.ts && bun run generate-schemas.ts ./openapi.json ./src/lib/validators.gen.ts",
-		"dev": "vite",
-		"check-ts": "tsc",
-		"build": "tsc && vite build",
-		"lint": "biome check src && prettier --check 'src/**/*.{s,}css'",
+		"setup:openapi": "bunx --bun openapi-typescript ./openapi.json -o ./src/lib/schema.gen.d.ts",
+		"setup:schemas": "bun run generate-schemas.ts ./openapi.json ./src/lib/validators.gen.ts",
+		"setup:all": "bun run setup:openapi && bun run setup:schemas",
+		"dev": "bun run setup:all && bun run vite",
+		"check-ts": "bun run tsc",
+		"build": "bun run tsc && bun run vite build",
+		"lint": "bun run biome check src && bun run prettier --check 'src/**/*.{s,}css'",
 		"preview": "vite preview"
 	},
 	"dependencies": {
```

Comment:
Can you double check if you removed all unused dependencies?

For one `json-refs` is still here, which is not used anymore in the new generator.